articles/openstack/ostack-how-manage-accounts-portal.md

```diff
@@ -143,16 +108,9 @@ To access OpenStack:
 
 ## Removing an OpenStack account via the UKCloud Portal
 
-If you want to remove an OpenStack user account, you must first revoke the user's permissions to all OpenStack projects before deleting the account.
-
-To remove an OpenStack account:
-
-1. Follow the steps in [Assigning and revoking OpenStack permissions](#assigning-and-revoking-openstack-permissions) to revoke the user's access permissions for all OpenStack projects.
-
-    > [!NOTE]
-    > It takes approximately five minutes for any changes you make within the UKCloud Portal to replicate to OpenStack.
+Removing a user's account in the portal will also remove their access to OpenStack.
 
-2. After waiting five minutes for the changes to replicate to OpenStack, return to the *Contacts* list, locate the user again and click the **Remove** button.
+1. Following the instructions above, navigate to the *Contacts* list, locate the user and click the **Remove** button.
```

Comment:
The section immediately above doesn't show how to get to the Contacts list - it's two sections above, so I'd be tempted to repeat the steps here (plus, if the reader just wants to delete a user and hasn't read the rest of the article, it's annoying to have to scroll up to find some steps, then scroll back down again to carry on with the remaining steps)

Reply:
Done.